scripts/exdump_monitor.sh

```diff
@@ -280,7 +280,7 @@ DTIM_latest_esiasi=-0.01
 DTIM_earliest_esatms=-1.50
 DTIM_latest_esatms=-0.51
 
-$ushscript_dump/bufr_dump_obs.sh $dumptime 0.50 1 sfcshp adpsfc rassda \
+$USHbufr_dump/bufr_dump_obs.sh $dumptime 0.50 1 sfcshp adpsfc rassda \
```

Comment:
Revert to using original variable, $ushscript_dump. It is already defined in JOBSPROC_DUMP_MONITOR.